event-sourcing/subscription-manager/src/main/java/uk/gov/justice/services/event/sourcing/subscription/manager/SubscriptionManagerProducer.java

```diff
@@ -37,31 +38,27 @@
     InterceptorChainProcessorProducer interceptorChainProcessorProducer;
 
     @Inject
-    SubscriptionDescriptorDefinitionRegistry subscriptionDescriptorRegistry;
+    SubscriptionsDescriptorsRegistry subscriptionDescriptorRegistry;
 
     @Inject
     QualifierAnnotationExtractor qualifierAnnotationExtractor;
 
     @Inject
     EventBufferSelector eventBufferSelector;
 
-    @Inject
-    Logger logger;
```

Comment:
Looks like a dodgy merge here...I think this is the correct way to define the logger

Reply:
updated